api/envoy/config/listener/v3/quic_config.proto

```diff
@@ -24,11 +24,17 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#protodoc-title: QUIC listener config]
 
 // Configuration specific to the UDP QUIC listener.
-// [#next-free-field: 12]
+// [#next-free-field: 13]
 message QuicProtocolOptions {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.api.v2.listener.QuicProtocolOptions";
 
+  message SaveCmsgConfig {
```

Comment:
Do you think the comment on the new field below suffices?

Reply:
I think it would be good to explain here what the fields mean. Just a brief description that this config specifies a specific level + type of CMSG data to save, or some such.

Reply:
Added.